src/runtime/string.go

```diff
@@ -205,6 +205,18 @@ func decodeUTF8(s string, index uintptr) (rune, uintptr) {
 	}
 }
 
+// indexByte returns the index of the first instance of c in the byte slice b,
+// or -1 if c is not present in the byte slice.
+//go:linkname indexByte internal/bytealg.IndexByte
+func indexByte(b []byte, c byte) int {
```

Comment:
This fails on Go 1.11 because `indexByte` is already defined in strings_go111.go. I would suggest simply renaming one of the two, for example you could rename the other one to `strings_indexByte` (that will be one of the things we'll drop when we drop Go 1.11 support so it's better to make that one look ugly instead of the function that we will likely keep for a longer time).